include/boost/geometry/algorithms/detail/disjoint/multipoint_geometry.hpp

```diff
@@ -63,12 +63,13 @@ namespace detail { namespace disjoint
 class multipoint_multipoint
 {
 private:
-    template <typename Iterator, typename CSTag>
+    template <typename Iterator, typename Strategy>
     class unary_not_disjoint_predicate
-        : geometry::less<void, -1, CSTag>
+        : geometry::less<void, -1, Strategy>
     {
     private:
-        typedef geometry::less<void, -1, CSTag> base_type;
+        //TODO: pass strategy
```

Comment:
Is this still TODO or already done? Because it is passed now. Or do you want to pass an instance? If so, can you rephrase the comment?

Reply:
This is done, removed now.